addon/mirage-graphql/resolvers/index.js

```diff
@@ -1,5 +1,15 @@
 import { GraphQLDateTime } from "graphql-iso-date";
 
 export default {
-  DateTime: GraphQLDateTime
+  DateTime: GraphQLDateTime,
+  // generic scalar serializes from a string to an object but deserializes as
+  // object for some reason ¯\_(ツ)_/¯. But hey, it works!
```

Comment:
```suggestion
  // object because of backend limitations.
```

Reply:
Resolved in 96f3bef43158aa98455210d342ee1dac9b07fb8e